packages/mobile/android/app/src/debug/AndroidManifest.xml

```diff
@@ -3,6 +3,7 @@
     xmlns:tools="http://schemas.android.com/tools">
 
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
```

Comment:
We may modify only `main` folder's manifest. `debug` one is intended to override some settings which we don't currently need

Reply:
updated